lib/kafka/offset_manager.rb

```diff
@@ -40,7 +40,7 @@ def commit_offsets
         @group.commit_offsets(@processed_offsets)
 
         @last_commit = Time.now
-        @processed_offsets.clear
```

Comment:
Think you should either clear `@committed_offsets` or merge `@processed_offsets` into `@committed_offsets.offsets`, b/c it now contains stale data.